src/lib/Libnet/net_server.c

```diff
@@ -592,10 +591,9 @@ wait_request(time_t waittime, void *priority_context)
 			}
 		}
 
+		int em_fd;
```

Comment:
As discussed this variable needs to be defined at the start of the function otherwise Windows build might mail.

Reply:
done